src/utils/wallet.js

```diff
@@ -218,17 +217,23 @@ export function WalletProvider({ children }) {
               )
             : new Account(
                 (() => {
-                  const { nonce, ciphertext } = privateKeyImports[
-                    walletSelector.importedPubkey
-                  ];
+                  const { nonce, ciphertext } =
+                    privateKeyImports[walletSelector.importedPubkey];
```

Comment:
What is `walletSelector.importedPubkey` and how is it different than the public key this PR is exposing?

Reply:
Sollet is using a library called `tweetnacl` which is used to derive a public/private keypair from the seedphrase. This is using `Ed25519` an algorithm that is not used for encryption. Also see here: https://crypto.stackexchange.com/questions/83266/why-are-ed25519-keys-not-recommended-for-encryption

Sollet is using specifically the Public-Key signatures feature, not the box features.

This PR is exporting a Public Key which can be used to encrypt content which then can be decrypted using the private key derived from the wallet seed. We are using a library called `eciesjs` which under the hood is using `secp256k1` which can be used for encryption/decryption.